<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Standard</class>
 <widget class="QWidget" name="Standard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="2">
      <widget class="QPushButton" name="btnNum6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="btnNum4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="btnInverse">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
	color:rgb(95, 95, 95);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);

}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>1/x</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="btnEqual">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(0, 95, 186);
}

QPushButton::hover{
	background-color: rgb(49, 125, 197);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(49, 125, 197);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btnSquare">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
	color:rgb(95, 95, 95);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);

}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>x²</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="btnNum2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="btnZhengFu">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>+/-</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="btnMinus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
	color:rgb(95, 95, 95);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);

}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btnPercentage">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
	color:rgb(95, 95, 95);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);

}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="btnNum5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="btnDivide">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
	color:rgb(95, 95, 95);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);

}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>÷</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btnDel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>⌫</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="btnNum7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btnNum9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="btnNum0">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btnSqrt">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
	color:rgb(95, 95, 95);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);

}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>2√x</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="btnNum8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="btnMultiple">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
	color:rgb(95, 95, 95);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);

}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>×</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="btnNum1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="btnPeriod">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="btnPlus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
	color:rgb(95, 95, 95);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);

}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="btnNum3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnClear">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>CE</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnClearAll">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	font: 30 18pt;
	background-color: rgb(252, 252, 252);
}

QPushButton::hover{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}

QPushButton::pressed{
	background-color: rgb(243, 243, 243);
	color:rgb(95, 95, 95);
}
</string>
       </property>
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLineEdit" name="display">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 63 20pt &quot;Bahnschrift SemiBold&quot;;
QLineEdit{
 color:#000;
}
</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
